src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -47,65 +47,51 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
 
             if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
```

Comment:
Can this if statement be removed? I don't think there is a scenario where this is not the case.

Reply:
Min version is 17763 which has UniversalApiContract 7

Reply:
I thought we were using 1809. We can keep this then 👍

Reply:
Yeah 1809 == 10.0.17763.0